src/Uno.UI/UI/Xaml/Internal/EventManager.uno.cs

```diff
@@ -70,6 +70,12 @@ public void RaiseLayoutUpdated()
 		// This is actually what we want and what WinUI does.
 		foreach (var item in _layoutUpdatedSubscribers)
 		{
+			// Sometimes, we are racing with GC and the DependencyObjectStore is disposed (via finalizer)
+			if (((IDependencyObjectStoreProvider)item.Key).Store.IsDisposed)
+			{
+				continue;
+			}
```

Comment:
This feels so hacky. It does the trick for now though.

cc @jeromelaban @ebariche